modules/doobie-hikari/src/main/scala/pureconfig/module/doobie/hikari/package.scala

```diff
@@ -0,0 +1,17 @@
+package pureconfig.module.doobie
+
+import _root_.doobie.enumerated._
+import _root_.doobie.hikari._
+
+import pureconfig._
+import pureconfig.generic.semiauto._
+
+package object hikari {
+
+  implicit lazy val readerTransactionIsolation: ConfigReader[TransactionIsolation] =
+    deriveEnumerationReader(ConfigFieldMapping(ScreamingSnakeCase, ScreamingSnakeCase))
+
+  implicit lazy val readerHikari: ConfigReader[Config] =
+    deriveReader
```

Comment:
Even though it's a very small performance penalty, do we benefit from lazy vals here (at the cost of the extra persistent memory)?

Reply:
I always implement implicits as `lazy val`s by habit without even thinking about it. That's also how Dotty/Scala 3 does it. Less thing to worry about, now I don't have to worry about it possibly not being initialized.

But if you have a preference, please let me know how else you'd like to have it, I'll be happy to accommodate you.